### PR TITLE
#3310 Update Invited Unregistered 'Join Project' Workflow

### PR DESCRIPTION
**Pull request by suppermancool**
#3310 Update Invited Unregistered 'Join Project' Workflow

### CODE REVIEW

File: src/projects/components/projectsCard/ProjectCard.jsx [hunk 1]
```diff
@@ -41,11 +42,29 @@ function ProjectCard({ project, duration, disabled, currentUser, history, onChan
...
-            </Link>
+            <div className="spacing join-btn-container">
+              {(!isAcceptingInvite) && (
+                <button
+                  onClick={(event) => {
+                    event.stopPropagation()
+                    callInviteRequest(project, true)
+                  }}
+                  className={'tc-btn tc-btn-primary tc-btn-md blue join-btn'}
+                  style={{margin: '5px'}}
```
Comment (maxceem) [inline src/projects/components/projectsCard/ProjectCard.jsx]: I know we use include style before, but can we avoid using inline CSS styles for new code and use SCSS files instead?

File: src/projects/components/projectsCard/ProjectCard.jsx [hunk 2]
```diff
@@ -41,11 +42,29 @@ function ProjectCard({ project, duration, disabled, currentUser, history, onChan
...
+                  Join project
+                </button>)}
+              {(!isAcceptingInvite) && (
+                <button
+                  onClick={(event) => {
+                    event.stopPropagation()
+                    callInviteRequest(project, false)
+                  }}
+                  className="decline-btn"
+                  style={{margin: '5px'}}
```
Comment (maxceem) [inline src/projects/components/projectsCard/ProjectCard.jsx]: I know we use include style before, but can we avoid using inline CSS styles for new code and use SCSS files instead?

File: src/projects/components/projectsCard/ProjectCard.jsx (hunk 1 above)
Comment (suppermancool) [reply src/projects/components/projectsCard/ProjectCard.jsx]: @maxceem done in latest commit

File: src/projects/components/projectsCard/ProjectCard.jsx (hunk 2 above)
Comment (suppermancool) [reply src/projects/components/projectsCard/ProjectCard.jsx]: @maxceem done in latest commit

**Comment by suppermancool** [comment]
@maxceem all done in latest commit

**Comment by suppermancool** [comment]
@maxceem all done in the latest commit

**Comment by suppermancool** [comment]
@maxceem i just removed the test code, pls check again, thanks

**Comment by suppermancool** [comment]
@maxceem done in the latest commit

**Pull request merged**